### DISCUSSION TITLE
Privacy Settings in NVDA

### DISCUSSION THREAD

**Discussion by tburc** [nvaccess/nvda#16930] [General]
Would people be open to Privacy Settings being added to the settings area of NVDA? I think it would be a good place to put the Screen Curtain and any new features that come up that could relate to privacy.

**Comment by CyrilleB79** [comment]
Yes, it makes sense. Maybe there is already an issue on this topic?

In addition I wonder if the following options would deserve a migration to a privacy panel:
* Allow NV Access to gather NVDA usage statistics
* maybe also Logging level

**Comment by CyrilleB79** [comment]
#16272See
